backend/src/main/kotlin/blue/mild/covid/vaxx/dto/response/IsinJobDtoOut.kt

```diff
@@ -0,0 +1,12 @@
+package blue.mild.covid.vaxx.dto.response
+
+data class IsinJobDtoOut(
+    var validatedPatientsSuccess: Int = 0,
```

Comment:
`var` v data class prosim ne, data classa jako takova by mela byt immutable, protoze jeji ucel je nest data z A do B, kdyz prochazi nekolika komponentama, tak by melo byt pripadne jasne, jak a kdy se meni aby se predeslo nejakemu mutable sdilenemu stavu

koukal jsem jak to pouzivas a v tom bode bych spis mel teda jednotlive hodnoty jako `var` a pak to hodil do data classy, ktera je ciste slozena jenom z `val`

tady se divim, ze neni nastvanej detekt, asi chybi pravidlo